vagrant/bootstrap.sh

```diff
@@ -51,8 +41,8 @@ sudo mysqladmin -p$MYSQL_ROOT_PASS create alliance_jabber
 sudo mysqladmin -p$MYSQL_ROOT_PASS create alliance_mumble
 sudo mysqladmin -p$MYSQL_ROOT_PASS create alliance_killboard
 
-sudo mysql -u root -p$MYSQL_ROOT_PASS -e "CREATE USER 'allianceauth'@'localhost' IDENTIFIED BY 'allianceauth'"
-sudo mysql -u root -p$MYSQL_ROOT_PASS -e "GRANT ALL PRIVILEGES ON * . * TO 'allianceauth'@'localhost'";
+sudo mysql -u root -p$MYSQL_ROOT_PASS -e "CREATE USER 'allianceserver'@'localhost' IDENTIFIED BY 'password'"
+sudo mysql -u root -p$MYSQL_ROOT_PASS -e "GRANT ALL PRIVILEGES ON * . * TO 'allianceserver'@'localhost'";
 sudo mysql -u root -p$MYSQL_ROOT_PASS -e "CREATE USER 'alliancemumble'@'localhost' IDENTIFIED BY 'alliancemumble'"
```

Comment:
we can do away with this user account as it's no longer the default in settings.py